src/Compilers/CSharp/Test/Semantic/Microsoft.CodeAnalysis.CSharp.Semantic.UnitTests.csproj

```diff
@@ -17,6 +17,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.CSharp" Version="$(MicrosoftCSharpVersion)" />
+    <PackageReference Include="Xunit.Combinatorial" Version="$(XunitCombinatorialVersion)" />
```

Comment:
><PackageReference Include="Xunit.Combinatorial" Version="$(XunitCombinatorialVersion)" /> [](start = 4, length = 89)

What prompted this change, it doesn't look like there are any source changes in this project?

Reply:
I can't find a specific reason for this change. I think this PR is going to need to be retargeted (contains a localization change), so I'll remove it if possible before the new build is kicked off.

Reply:
➡️ The test TestReportingDiagnosticWithInvalidLocation uses this.